electron-component/main.js

```diff
@@ -0,0 +1,35 @@
+const WindowsManager = require("./components/WindowsManager");
```

Comment:
the structure of this file is not good
it should be
1. requires
2. call to the main function
3. function in the order of execution

for example:
```javascript
const { app } = require('electron');

init();

function init() {
   a();
   c();
}

function a() {
  ...
  b()
  ...
}

function b() {
 ...
}

function c() {
  ...
}
```

Reply:
done
the way you wrote it may cause hoisting in some cases - so I did it - just in different order.